packages/taro-cli/src/create/project.ts

```
@@ -126,6 +127,14 @@ export default class Project extends Creator {
     await this.askTemplate(conf, prompts, templates)
     const templateChoiceAnswer = await inquirer.prompt<IProjectConf>(prompts)
 
+    // 导航步骤扩展
+    try {
+      if (typeof conf.ask === 'function') {
+        const { ask, ...other } = conf
```

ESLint

[error] 'ask' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)